Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs

```diff
@@ -67,15 +67,23 @@ public static class AdvancedFluentLayoutExtensions
         public static IEnumerable<FluentLayout> FullWidthOf(this UIView view, UIView parent, nfloat? margin = null)
         {
 			var marginValue = margin.GetValueOrDefault(DefaultMargin);
-			yield return view.Left().EqualTo().LeftOf(parent).Plus(marginValue).WithIdentifier("Left");
```

Comment:
I found out `yield return` will interfere with constants manipulation and they won´t work. For instance:
```
var layouts = view.FullWidthOf(whatever);
layouts.GetLayoutById("Top").Constant = 200 // this didn´t work until I removed yield return
```